### PR TITLE
I2 30 ordershow

### PR DESCRIPTION
**Pull request by dhalverson**
Adds
- Cancel order test
- Destroy action in orders controller
- Cancel button in cart show
- Updates order time to local time zone 
- Updates order profile page to include item order status
- Adds status and default of processing to item orders table

This may have to be updated down the line to work with some of the other stories, where we decrement the inventory once and order is fulfilled. I also noticed that the cancel order is still visible on orders that have been cancelled already, I would like to pair up possibly and work through adding in some if logic, outside of the story maybe, to make this work better.

### CODE REVIEW

**Comment by dhalverson** [comment]
Ambiguous match, found 2 elements matching visible link or button "My Orders"
     # ./spec/features/users/user/show_spec.rb:83:in `block (4 levels) in <top (required)>'

Also has an error, Daniel mentioned this has been resolved in another PR so I left it in mine

File: spec/features/orders/show_spec.rb [hunk 1]
```diff
@@ -48,5 +48,61 @@
...
+    end
+
+    it "The order itself is given a status of cancelled" do
+      visit("/profile/orders/#{@user_1.orders.first.id}")
+      click_button("Cancel Order")
+      expect(page).to have_content("Your order has been cancelled!")
+
+    end
+
+    it "Any item quantities in the order have their quantities returned to their respective values" do
```
Comment (dhalverson) [inline spec/features/orders/show_spec.rb:85]: Any suggestions on how to test the quantity here?

File: app/controllers/orders_controller.rb [hunk 2]
```diff
@@ -26,8 +26,25 @@ def create
...
+  def destroy
+    order = Order.find(params[:id])
+    if order.save
+      order.item_orders.each do |item_order|
+        item = Item.find(item_order.item_id)
+        item.inventory += item_order.quantity
+        item.save
+        item_order.update(status: "unfulfilled")
+        item_order.save
+      end
```
Comment (PhilipDeFraties) [inline app/controllers/orders_controller.rb]: are the saves necessary?

**Review by D-Lessenden — APPROVED** [review]
Everything looks good to me. I didn't merge because Phil's comment was unaddressed.

**Review by dhalverson — COMMENTED** [review]
Updates have been added, happy with testing for now, until we actually start updating inventory once an order is completed

File: app/controllers/orders_controller.rb (hunk 2 above)
Comment (dhalverson) [reply app/controllers/orders_controller.rb]: These have been removed per testing!

**Review by PhilipDeFraties — APPROVED** [review]

**Comment by PhilipDeFraties** [comment]
Looks good @dhalverson

**Pull request merged**